### DISCUSSION TITLE
Hydration failed initial UI : Password Manager

### DISCUSSION THREAD

**Discussion by danielschmid** [vercel/next.js#57623] [Help]
### Summary

I get a lot of Hydration Error messages due to the fact that Password Managers inject code to my page. What would be a good workaround for that problem ?

**Comment by vignesh-gupta** [comment]
Hey @danielschmid, We have seen alot of issue because of the chrome extension but I always suggest if you are not using that extension much, remove it. And I think browsers have inbuild password manager. So you can use that.

**Reply by danielschmid** [reply]
Thanks for your resonse, i was more concerned about what other users would experience. Suppressing the hydration warning on the `form` does suppress the error message most of the time but strangely enough not always.
